config/imagery/tauranga_urban_2017_0-1m.json

```diff
@@ -0,0 +1,14 @@
+{
+  "type": "raster",
+  "id": "ts_tauranga_urban_2017_0-1m",
```

Comment:
I cannot view this record in basemaps?

```
https://basemaps.linz.govt.nz/?i=tauranga-urban-2017-0.1m # Current
https://basemaps.linz.govt.nz/?i=tauranga_urban_2017_0-1m # basemaps-config
```

Reply:
I wonder which is in actually in use?

Reply:
We want to format all the names from `tauranga_urban_2017_0-1m` to `tauranga-urban-2017-0.1m`